snorql-framework/pom.xml

```diff
@@ -63,6 +63,39 @@
             <version>3.6.0</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.quartz-scheduler</groupId>
+            <artifactId>quartz</artifactId>
+            <version>2.3.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.quartz-scheduler</groupId>
+            <artifactId>quartz-jobs</artifactId>
+            <version>2.3.2</version>
+        </dependency>
+        <dependency>
+            <groupId>com.microsoft.sqlserver</groupId>
+            <artifactId>mssql-jdbc</artifactId>
```

Comment:
why is this required?

Reply:
We should ideally not be mandating including any specific db driver in `snorql-framework`

Reply:
Post testing confirmed that Quartz does not require a JDBC driver explicitly for which this dependency was intended.
Removed mssql-jdbc dependency